cms/static/sass/views/_settings.scss

```diff
@@ -271,6 +271,17 @@
             display: inline-block;
           }
         }
+
+        input[type="radio"] {
```

Comment:
instead of using an input selector here could you use a class? I know this isn't the same as the pattern for the elements above and below but it should make things a bit easier to organize and parse eventually. Similarly, using the relevant class names for the & + label selector below would be preferred. (Even though again, I know we used that pattern for settings above this one.)